source/plugin/scheduler.rst

```diff
@@ -93,6 +114,34 @@ To cancel a task, simply call the ``Task#cancel`` method:
 
 	task.cancel();
 
+If you need to cancel the task from within the runnable itself, you can instead opt to use a `Consumer<Task>` in order to
+access the task. The below example will schedule a task that will count down from 60 and cancel itself upon reaching 0.
+
+.. code-block:: java
+
+        Scheduler scheduler = Sponge.getScheduler();
+        Task.Builder taskBuilder = scheduler.createTaskBuilder();
+
+        Task task = taskBuilder.execute(new CancellingTimerTask())
+            .interval(1, TimeUnit.SECONDS)
+            .name("Self-Cancelling Timer Task").submit(plugin);
+    }
```

Comment:
this brace seems superfluous, since it has no matching opening brace. either wrap the above in a function or remove the brace